recipes/trajan/LICENSE

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
You do not need to include this copy of the license file since it is included in the source distribution.

Reply:
Seems to be necessary, according to linter.

Reply:
You should be able to remove the file, but you would need to keep `license_file: LICENSE` in the `meta.yaml`

Reply:
Ah ok, thanks.